nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1503,16 +1503,14 @@ private Description handleInvocation(
         handler.onOverrideMethodInvocationParametersNullability(
             state.context, methodSymbol, isMethodAnnotated, argumentPositionNullness);
 
-    ImmutableSet<Integer> nonNullPositions =
-        argumentPositionNullness.entrySet().stream()
```

Comment:
This stream computation was showing up a bit in profiles, so I changed the code to not construct `nonNullPositions` explicitly